en/simulation/gazebo.md

```diff
@@ -1,6 +1,10 @@
 # Gazebo Simulation
 
-[Gazebo](http://gazebosim.org) is a 3D simulation environment for autonomous robots. It supports standalone use (without ROS) or SITL + ROS.
+[Gazebo](http://gazebosim.org) is a powerful 3D simulation environment for autonomous robots that is particularly suitable for testing object-avoidance and computer vision. It can also be used for [multi-vehicle simulation](../simulation/multi-vehicle-simulation.md) and is commonly used with [ROS](../simulation/ros_interface.md), a collection of tools for automating vehicle control. 
```

Comment:
FYI, just a better introduction and also cross linking back to the parent doc for "general simulator info".

Reply:
> a collection of tools for automating vehicle control.

not just that, but I guess it fits the purpose.